charms/dex-auth/reactive/dex_auth.py

```diff
@@ -56,10 +58,7 @@ def start_charm():
     port = hookenv.config("port")
     public_url = hookenv.config("public-url")
 
-    oidc_client_info = endpoint_from_name('oidc-client').get_config()
-    if not oidc_client_info:
-        layer.status.blocked("No OIDC client information found")
-        return False
+    oidc_client_info = endpoint_from_name('oidc-client').get_config() or []
```

Comment:
This has the potential to raise a "NoneType does not have attribute get_config" error if the relation doesn't exist.

```suggestion
    oidc_client = endpoint_from_name('oidc-client')
    oidc_client_info = oidc_client.get_config() if oidc_client else []
```